src/tests/unit/tests/bug-filing/common/http-query-builder.test.ts

```diff
@@ -46,20 +46,20 @@ describe('HTTPQueryBuilder', () => {
     });
 
     describe('truncate urls', () => {
-        const actualUrl2990 = repeat('<-10->', 299);
+        const actualUrl2990 = repeat('<-10->', 199);
```

Comment:
Rename the variables to match new lengths? (this was originally a URL with 2990 characters)

Reply:
Updated